implementations/js-hyperjump/bowtie_hyperjump.js

```diff
@@ -1,12 +1,10 @@
-const path = require("path");
-const readline = require("readline/promises");
+import * as path from "path";
+import readline from "readline/promises";
+import { addSchema, validate } from "@hyperjump/json-schema";
```

Comment:
> I can't tell if there's still a way to get all dialects imported at once

In this version you need to be explicit about which dialects you want support for.

```suggestion
import { addSchema, validate } from "@hyperjump/json-schema/draft-2020-12";
import "@hyperjump/json-schema/draft-2019-09";
import "@hyperjump/json-schema/draft-07";
import "@hyperjump/json-schema/draft-06";
import "@hyperjump/json-schema/draft-04";
```

Reply:
Ah I see, thanks! I saw that in your docs (that one should import `addSchema` and friends from the relevant version) but didn't realize it meant the `imports` themselves enable the draft even *without* doing so. Thanks, this indeed seems to work.